src/modules/ListedCompany/utils/chartTime.ts

```
@@ -0,0 +1,75 @@
+}
+
+export function formatChartTooltip(row: ChartRecordTuple): string {
+  const date = normalizeDate(row[0])
+  return `${date} | 交易時間：${row[1]} | 開盤價: ${row[2]} | 最高價: ${row[3]} | 最低價: ${row[4]} | 目前成交價: ${row[5]} | 成交量: ${row[6]}`
+}
+
+export function formatIntradayTooltip(point: ParsedChartPoint): string {
+  const time = normalizeTime(point.raw[1]).slice(0, 5)
+  const price = Number.isFinite(point.close) ? point.close.toFixed(1) : '—'
+  const volume = Number.isFinite(point.volume) ? String(Math.round(point.volume)) : '—'
+  return `時間 ${time}　價格 ${price}　成交量 ${volume}`
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)